packages/nextjs/app/builders/[0x2a2cC957b37159e844c1AD8E7C662568e855C79D]/page.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import type { NextPage } from "next";
+import { normalize } from "viem/ens";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { CodeBracketIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+import { useScaffoldReadContract } from "~~/hooks/scaffold-eth";
+
+const PacePhilpotProfile: NextPage = () => {
+  const builderAddress = "0x2a2cC957b37159e844c1AD8E7C662568e855C79D";
```

Comment:
Since this is a constant, it makes sense to place it outside of the component. So it's created only once at module initialization, rather than every render.